### DISCUSSION TITLE
getServerSideProps ECONNREFUSED when in docker

### DISCUSSION THREAD

**Discussion by stasyaner** [vercel/next.js#17585] [Help] [answered]
Hi the community,

I have a getServerSideProps function which calls my graphql backend. It works fine locally (although I had some mysterious issues at the beginning #17437), but when I deploy my app into docker, the call fails with ECONNREFUSED. It never reaches my backend. Full error: `request to http://localhost:3415/graphql/ failed, reason: connect ECONNREFUSED 127.0.0.1:3415`. Appreciate your help. Thank you.

**Comment by leerob** [comment]
It's hard to know what the issue is without a reproducible example. Could you share a CodeSandbox link or some more code?

**Comment by stasyaner** [comment, marked as answer]
Thank you for your help. The issue was you can couldn't call docker host machine on a bridged network

**Reply by 2qxtx** [reply]
How did you get around this?
